en/sensor/pmw3901.md

```diff
@@ -107,8 +109,8 @@ Pixhawk SPI Port (from left to right) | Bitcraze flow board
 6 (CS2) | CS
 7 (GND) | GND
 
-In order to connect the bitcraze flow board to the Pixhawk you will need to solder the wires of the Pixhawk SPI cable to the flow board.
-An SPI cable has 7 wires, from which you need to connect 6 to the flow board.
+To connect the bitcraze flow board to the Pixhawk, you will need to solder the wires of the Pixhawk SPI cable to the flow board.
+An SPI cable has seven wires, from which you need to connect 6 to the flow board.
```

Comment:
Why change 7 to seven and not 6 to six? Personally I prefer the use of numbers EVEN though using the word is technically more correct. It is more concise and easier to read.